### PR TITLE
Range.createContextualFragment is not overriden (close #535)

### LINKED ISSUE #535: Range.createContextualFragment is not overriden

### PR DESCRIPTION
**Pull request by churkin**
\cc @AlexanderMoskovkin @miherlosev

### CODE REVIEW

File: src/client/sandbox/node/window.js [hunk 1]
```diff
@@ -183,6 +184,17 @@ export default class WindowSandbox extends SandboxBase {
             };
         }
 
+        window.Range.prototype.createContextualFragment = function () {
```
Comment (miherlosev) [inline src/client/sandbox/node/window.js]: It is wrong. In 9 `window.Range.prototype` does not contains method `createContextualFragment`.
Need to change on:
```javascript
if (window.Range.prototype.createContextualFragment) {
    window.Range.prototype.createContextualFragment = function (){...}
}
```

**Comment by miherlosev** [comment]
\r-

**Comment by churkin** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**